documentation/decisions/013_docs_content_styleguide.md

```diff
@@ -0,0 +1,54 @@
+---
+status: Pending
+date: 2023-09-01
+deciders: nwithan8
+---
+
+### Inter-section linking
+
+- Any reference to an EasyPost object structure (e.g. `Address`, `Shipment`, `Order`) is linked to the corresponding section.
+  - References to an EasyPost object as a type in a definition table (e.g. the `to_address` field of a `Shipment` is an `Address`) are linked to that object's definition table (e.g. the `Address` object definition table on the Address page).
+  - References to an EasyPost object in a text paragraph (e.g. "Check out the `to_address` field.") are linked to the API documentation section for that object (e.g. the top of the Address page).
+
```

Comment:
question: why the extra lines? Can we kill them to keep the bullets flowing together?